tools/common/Frameworks.cs

```diff
@@ -267,9 +269,9 @@ public static Frameworks GetiOSFrameworks (Application app)
 
 				{ "ARKit", "ARKit", 11 },
 				{ "CoreNFC", "CoreNFC", 11 },
-				{ "DeviceCheck", "DeviceCheck", 11 },
+				{ "DeviceCheck", "DeviceCheck", new Version (11, 0), new Version (int.MaxValue, int.MaxValue) /* no headers provided for simulator */ },
```

Comment:
let's create a constant to ease readability (and avoid unrequited extra instances), e.g.

```
// used when headers for a framework is not provided for the simulator
static Version NotAvailableInSimulator = new Version (int.MaxValue, int.MaxValue)'
```

Reply:
Also this file is now included in tests so that new constant can be used to create a canary test - one that will tell us if/when a framework becomes available in a newer version of Xcode.

Another good one (it happened before) would be to detect the others framework are present, like we expect them (e.g. if one gets removed by mistake).

Reply:
> let's create a constant to ease readability (and avoid unrequited extra instances)

Fixed.

> Also this file is now included in tests so that new constant can be used to create a canary test - one that will tell us if/when a framework becomes available in a newer version of Xcode.
> 
> Another good one (it happened before) would be to detect the others framework are present, like we expect them (e.g. if one gets removed by mistake).

I think this can be done in a separate PR.